www/howitworks.json

```diff
@@ -23,7 +23,7 @@
         "classes": "xs-hide sm-hide",
         "href": "/"
       },
-      "name": "The Blog",
+      "name": "AMP start home logo",
```

Comment:
Could you provide proper text for this then? The Templates and things usually have random logos.